actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -337,3 +337,128 @@ fn effective_gas_limit<RT: Runtime>(system: &System<RT>, gas: U256) -> u64 {
     let gas = gas.to_u64_saturating();
     std::cmp::min(gas, gas_rsvp)
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::evm_unit_test;
+    use fil_actors_evm_shared::uints::U256;
+
+    #[test]
+    fn test_calldataload() {
+        evm_unit_test! {
+            (m) {
+                CALLDATALOAD;
+            }
+            m.state.input_data = vec![0x00, 0x01, 0x02].into();
+            m.state.stack.push(U256::from(1)).unwrap();
+            let result = m.step();
+            assert!(result.is_ok(), "execution step failed");
+            assert_eq!(m.state.stack.len(), 1);
+            assert_eq!(m.state.stack.pop().unwrap(), U256::from(0x0102) << 240);
+        };
+    }
+
+    #[test]
+    fn test_calldataload2() {
```

Comment:
Let's give these actual names with meaning. And/or add comments explaining _what_ we're testing and why.